Automattic/newspack-network
# feat: handle user deletion event on the network

### All Submissions:






### Changes proposed in this Pull Request:



Adds user deletion handling.

(Note: this is a part of integrating the changes from the `data-integrity-improvements` branch (https://github.com/Automattic/newspack-network/pull/89))

### How to test the changes in this Pull Request:

1. Register a user on one of the sites, wait for the data to sync*
2. Delete the user on one of the sites, wait for the data to sync
3. Observe the user was deleted from all sites
4. Observe there is only one `network_user_deleted` event in the event log

\* can be sped up by running `wp newspack-network process-webhooks` on the node and then `wp newspack-network sync-all` on the other node sites

### Other information:
